Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/MicrosoftGraphMail.py

```diff
@@ -916,7 +918,7 @@ def _parse_email_as_incident(self, email):
         parsed_email = self._parse_item_as_dict(email)
 
         # handling attachments of fetched email
-        attachments = self._get_email_attachments(message_id=email.get('id', ''))
+        attachments = self._get_email_attachments(message_id=email.get('id', ''), overwrite_rate_limit_retry=True)
```

Comment:
should this be true here?

Reply:
oops your'e right should be the given argument. I'll fix it